src/executor.rs

```diff
@@ -11,13 +11,13 @@
 //!
 //! Run a single-threaded and a multi-threaded executor at the same time:
 //!
-//! ```
+//! ```no_run
```

Comment:
This is the main example at the top level of the framework.

It should not be marked as no_run. Maybe we can demonstrate it with std::thread creating a simple thread.
`shutdown()` will also obviously not work. This is a good candidate for us to just use our internal `Timer` future.

Reply:
This was changed in e4955970a4a2b85427f537df42d595d7a79082bc in favor of a simpler example that uses `std::thread`s.